cmd/blockchaincmd/convert.go

```diff
@@ -268,6 +267,7 @@ func InitializeValidatorManager(
 	validatorManagerAddrStr string,
 	proxyContractOwner string,
 	useACP99 bool,
+	signatureAggregatorFlags flags.SignatureAggregatorFlags,
```

Comment:
we need to restrict passing in this flags objects, the limit is the CLI/SDK frontier, but probably we should also
avoid passing in flags objects outside of cmd helpers.